features/environment.py

```diff
@@ -31,7 +38,8 @@ def before_scenario(context, scenario):
         return
 
 
-def clean_db(context, database="test"):
+def prepared_db(context, setup_files=CLEANUP_FILES, database="test"):
```

Comment:
`prepare_db` instead of `prepare**d**_db`?

Reply:
yup, you are right, lemme fix it

Reply:
fixed